### DISCUSSION TITLE
Email on server creation

### DISCUSSION THREAD

**Discussion by csgo-s** [pterodactyl/panel#3058] [Support] [answered]
Hi
Pterodactyl doesn't send email to clients on server creation
only sends email on account creation
what is the problem?

**Comment by sinjs** [comment]
Pterodactyl doesn't send emails on server creation. 
There is no problem, that is completly normal.

**Reply by trenutoo** [reply]
> Pterodactyl doesn't send emails on server creation.
> There is no problem, that is completly normal.

Panel sends emails for the following reasons:

Account created, Server installed, Added/removed from a server as a sub user, and forgot password emails. You can see and customize them in the app/Notifications folder.

**Comment by trenutoo** [comment, marked as answer]
This has been fixed in the latest 1.3 release
